### PR TITLE
translate: cluster-level resource propagation pause and resume capabilities

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Add Chinese version of #668 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md [hunk 1]
```diff
@@ -671,3 +671,74 @@ spec:
 ```
 The `nginx` deployment in `default` namespace will be propagated to cluster `member2`.
 
+
+## 资源分发的暂停与恢复
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md:675]:
```suggestion
## 暂停与恢复资源分发
```
Heandings can have a consisten style of `v+n`

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md [hunk 2]
```diff
@@ -671,3 +671,74 @@ spec:
...
+  #...
+  suspension:
+    dispatching: true
+```
+
+在 Karmada 控制平面上对 `nginx` Deployment 的更新将不会被同步到任何成员集群。
+
+### 暂停向特定集群分发资源
+
+为了暂停向个别的成员集群分发资源，您可以在 `.spec.suspension.clusterNames` 字段中执行集群名称：
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md:704]:
```suggestion
为了暂停向个别的成员集群分发资源，您可以在 `.spec.suspension.clusterNames` 字段中指定集群名称：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md [hunk 3]
```diff
@@ -671,3 +671,74 @@ spec:
...
+spec:
+ #...
+ placement:
+  clusterAffinity:
+   exclude:
+    - member1
+    - member3
+  #...
+  suspension:
+    clusterNames: ["member3"]
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md:720]:
```suggestion
    clusterNames: ["member3"]   # 修改此处的集群名称
```
Seems this kind of comment is easier to identify.

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md:675]: Thanks for the clear instructions.

File: docs/userguide/scheduling/resource-propagating.md [hunk 4]
```diff
@@ -671,14 +671,13 @@ spec:
...
 
-
 ## Suspend and resume of resource propagation
 
 The `.spec.suspension` field in the `PropagationPolicy` and `ClusterPropagationPolicy` allows suspending and resuming resource propagation to one or more clusters.
 
 ### Suspend resource propagation to all clusters
 
-To suspend resource propagfation work to all member clusters, use the following configuration:
+To suspend resource propagation work to all member clusters, use the following configuration:
```
Comment (RainbowMango) [inline docs/userguide/scheduling/resource-propagating.md:680]: nice finding.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**